### DISCUSSION TITLE
[Web] Styling actions do not work over multiple paragraphs (more than one new line)

### DISCUSSION THREAD

**Discussion by vidklopcic** [singerdmx/flutter-quill#591] [Web]
Using the latest quill (1.9.0) toggling styles (bold, italic, etc.) over multiple paragraphs doesn't seem to function correctly. I can apply the style, but cannot turn it off (by toggling the same button).

https://user-images.githubusercontent.com/8578790/128711382-f434d7ca-25a5-43e0-b01c-8e938ef47159.mov

**Comment by florianh01** [comment]
Can reproduce on mobile when there is an empty line between two paragraphs

**Comment by singerdmx** [comment]
@florianh01 could you work on a fix, please? I know you can find the fix :)

**Comment by florianh01** [comment]
I can, but it will take some time

**Comment by vidklopcic** [comment]
Probably related - here is another way to break the styling toggle button:

https://user-images.githubusercontent.com/8578790/128829878-7dd1fb4f-b3c2-4d27-ac8e-3a5763f83cf3.mov
